tensorboard/backend/event_processing/event_accumulator.py

```diff
@@ -587,7 +601,25 @@ def _ProcessScalar(self, tag, wall_time, step, scalar):
 
   def _ProcessTensor(self, tag, wall_time, step, tensor):
     tv = TensorEvent(wall_time=wall_time, step=step, tensor_proto=tensor)
-    self.tensors.AddItem(tag, tv)
+    with self._tensors_by_tag_lock:
+      if tag not in self.tensors_by_tag:
+        reservoir_size = self._GetTensorReservoirSize(tag)
+        self.tensors_by_tag[tag] = reservoir.Reservoir(reservoir_size)
+    self.tensors_by_tag[tag].AddItem(_TENSOR_RESERVOIR_KEY, tv)
+
+  def _GetTensorReservoirSize(self, tag):
+    default = self._size_guidance[TENSORS]
+    summary_metadata = self.summary_metadata.get(tag)
+    if summary_metadata is None:
+      return default
+    # In the vast majority of cases, there will not be more than one
```

Comment:
I think that was what I want to articulate as well - no summary op adds data for 2 plugins.